etc/downloads_badges.py

```diff
@@ -1,19 +1,12 @@
-"""
-This code is meant to be run on Zapier
-
-It is used to add the `downloads/week` badge to our README
-"""
 from datetime import date, timedelta
 import json
 import math
 import os
 import requests
 
 
-# This function is duplicated from `zapier_total_downloads`. This is intentional,
-# because Zapier only has access to one script at a time, so we cannot import
 def millify(n):
-    """Abreviate a number to nearest thousand, million, etc.
+    """Abbreviate a number to nearest thousand, million, etc.
```

Comment:
👏

Reply:
Can't believe that got approved the first time, smh